app/DoctrineMigrations/Version20180101010000.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Squashed migration that replaces all previous migrations.
+ *
+ * The problem with the previous migrations was that there was no initial migration
+ * to base the other migrations on. This is fixed with this migration.
+ *
+ * This migration is the result of bin/console doctrine:schema:create --dump-sql
+ */
+class Version20180101010000 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf(
+            $this->connection->getDatabasePlatform()->getName() !== 'mysql',
+            'Migration can only be executed safely on \'mysql\'.'
+        );
+
```

Comment:
Maybe it's an idea to cleanup the migrations table to prevent warnings when running the doctrine commands.
   ```
     // If this migration is run on an existing DB just clear the version table and assume we are up-to-date
        if (count($this->version->getConfiguration()->getMigratedVersions())) {
            $this->addSql("TRUNCATE migration_versions");
            return;
        }
```

Reply:
A good suggestion and thanks for the discussion IRL.